src/icons/BorderStyleDashed.tsx

```diff
@@ -0,0 +1,16 @@
+import Box from "@mui/material/Box";
+import { MENU_BUTTON_FONT_SIZE_DEFAULT } from "../controls/MenuButton";
+
+function BorderStyleDashed() {
+  return (
+    <Box
+      sx={(theme) => ({
+        border: `3px dashed ${theme.palette.text.primary}`,
+        width: MENU_BUTTON_FONT_SIZE_DEFAULT,
+        height: MENU_BUTTON_FONT_SIZE_DEFAULT,
```

Comment:
It looks like these new border style icons appear a bit larger than the rest of the standard MUI icons. Should these be smaller?